lte/gateway/docker/docker-compose.yaml

```diff
@@ -161,6 +174,8 @@ services:
       retries: 3
     cap_add:
       - NET_RAW
+    environment:
+      MAGMA_DEV_MODE: ${MAGMA_DEV_MODE}
     command: /usr/bin/env python3 -m magma.monitord.main
 
   redirectd:
```

Comment:
Is there a reason it isn't needed for redirectd?

Reply:
Its probably not needed for a lot of other service as well. But I only removed it from service where I know from the top of my head it wasn't required.